src/Components/App.tsx

```diff
@@ -33,7 +34,8 @@ function App():ReactElement{
       {/* {token ? <Drawer /> : null} */}
       {isLoggedIn ? <Drawer /> : null}
       <Switch>
-        <Route path="/" component={LoginPage} exact />
+        {/* <Route path="/" component={LoginPage} exact /> */}
+        <Route path="/" component={ResetPassword} exact />
```

Comment:
Let's not override the "/" path. Make a new route, maybe called "/reset" or something. Right now it looks like the application will no longer show the login page.